docs/src/developers_guide/contributing_ci_tests.rst

```diff
@@ -92,12 +92,11 @@ Iris target branch by a core developer.
 
 .. _testing_cla:
 
-SciTools CLA Checker
-********************
+`CLA Assistant`_
+****************
 
-A bot which checks that the GitHub author of the pull-request has signed the
-**SciTools Contributor's License Agreement (CLA)**.  For more information on
-this please see https://scitools.org.uk/organisation.html#governance.
```

Comment:
I've chosen to remove reliance on the scitools.org.uk governance docs, since our repo admin has evolved in ways that make it less relevant, and our current development culture often cuts scitools.org.uk out of the loop.

Happy to be argued against.